with-nativewind/README.md

```diff
@@ -20,4 +22,4 @@
 
 <!-- Link to related Expo or library docs -->
 
-- This example is using the [`nativewind`](https://www.nativewind.dev/) library which lets you use Tailwind CSS in react-native.
+- This example replicates the [NativeWind Expo Router setup instructions](https://www.nativewind.dev/getting-started/expo-router) for NativeWind v.4.
```

Comment:
```suggestion
- This example replicates the [NativeWind Expo Router setup instructions](https://www.nativewind.dev/getting-started/expo-router) for NativeWind v4.
```

I think it can be omitted. A common pattern is to just say vX (X being the number).